### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -895,21 +895,27 @@ msgid ""
 "sparse-checkout definition but are not sparse due to local\n"
 "modifications.\n"
 msgstr ""
+"Наступні шляхи, що було винесено за межі визначення часткового\n"
+"переходу, не є частковими через локальні\n"
+"Зміни.\n"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"зміни.\n"
```

File: po/uk.po [hunk 2]
```diff
@@ -895,21 +895,27 @@ msgid ""
...
 
 #: advice.c:290
 msgid ""
 "To correct the sparsity of these paths, do the following:\n"
 "* Use \"git add --sparse <paths>\" to update the index\n"
 "* Use \"git sparse-checkout reapply\" to apply the sparsity rules"
 msgstr ""
+"Щоб виправити частковість цих шляхів, виконайте наступне:\n"
+"* Використайте \\”git add --sparse <шляхи>\\” для оновлення індексу\n"
+"* Використайте \\”git sparse-checkout reapply”, щоб застосувати правила частковості"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"* Використайте \"git sparse-checkout reapply\", щоб застосувати правила частковості"
```

**Review by kasya — COMMENTED** [review]
LGTM!

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po]: Right 👍

**Pull request merged**